x-pack/plugins/security/public/account_management/account_management_page.tsx

```diff
@@ -5,20 +5,24 @@
  */
 import { EuiPage, EuiPageBody, EuiPanel, EuiSpacer, EuiText } from '@elastic/eui';
 import React, { useEffect, useState } from 'react';
-import { SecurityPluginSetup } from '../../../../../../../plugins/security/public';
-import { getUserDisplayName, AuthenticatedUser } from '../../../../common/model';
+import { NotificationsStart } from 'src/core/public';
+import { getUserDisplayName, AuthenticatedUser } from '../../common/model';
+import { AuthenticationServiceSetup } from '../authentication';
 import { ChangePassword } from './change_password';
+import { UserAPIClient } from '../management';
 import { PersonalInfo } from './personal_info';
 
 interface Props {
-  securitySetup: SecurityPluginSetup;
+  authc: AuthenticationServiceSetup;
+  apiClient: PublicMethodsOf<UserAPIClient>;
```

Comment:
__note:__ we'll get rid of these `PublicMethodsOf` approximately at the same time this issue is being handled in the core https://github.com/elastic/kibana/issues/54906